README.md

```diff
@@ -41,37 +44,77 @@ For rates defined as 30/1.001 or 24/1.001, system implementers typically use rat
 
 Another less common approach in the community is to use what we refer to as a "factorized integer" - an integer that is an index over an implicit rate.  The rate is a large number that is a common factor of a common media rates.  See [^8] and [^9].  These are bounded by both by integer limits, especially for coarser rates and also cannot handle non-integer multiplication or rates whose factors are not present in their rate.  We've included them in the data limits table.
 
+### Sampling and Interpolation
+
+These experiments demonstrate, for a variety of representations, the numeric limits at which a time ordinate unmistakably corresponds to a particular frame at a given framerate. In the regime close to these limits, accurate subsampling and interpolation are not possible due to the fact that the numbers are at the limits of their representation. However, a consequence of the fact that the ordinates are distinguishable at rate means that they can be reversibly be moved to the origin and back again. So, a system that must perform sampling and interpolation may reliably do so by translating a sequence of interest close to the origin and sampling accurately there, as long as it make sense to do the sampling or interpolation in a local frame of reference.
+
+### Continuously Variable Framerate
+
+Continuously variable framerate (CVFR) in modern smartphone cameras operates on a discrete sensor readout clock, typically at a high frequency (e.g., 240 Hz in some configurations). However, the output framerate is not achieved by simple frame skipping; rather, it is dynamically controlled through a combination of exposure time, gain, and readout timing adjustments.
+
+Rather than continuously recording at a fixed high framerate and discarding frames, the system dynamically adjusts exposure duration and readout intervals to achieve a desired output framerate while maintaining synchronization with the rolling shutter scan. This allows for adaptive framerate changes in response to lighting conditions, motion, or energy efficiency considerations.
+
+Frames can be synthesized at the target rate through selective readout, integration, and interpolation techniques, rather than merely dropping frames from a fixed high-rate sequence. This approach optimizes image quality while allowing for framerate flexibility within the constraints of the sensor's rolling shutter and exposure control mechanisms.
+
+Since CVFR systems operate on a discrete readout clock, the precision analysis discussed earlier applies directly. The readout timing and exposure adjustments introduce constraints on exact frame alignment, meaning precision limits are dictated by the granularity of the underlying clock. As with other time representations, cumulative errors may emerge when approximations or truncations occur over extended sequences. Additionally, factors such as sensor clock jitter and temperature-induced timing variations should be considered in long-duration recording scenarios.
```

Comment:
I haven't deal with CVFR stuff before, so apologies for the naive question . I'm assuming here that the CVFR treats audio and picture separately, and that presumably the fastest the CVFR can run at is that underlying clock (so 240hz mentioned previously for example).  Frames can vary in length by some number of 240 hz ticks so that they always line up at the 240 tick boundaries even if exactly how many of the 240 ticks a frame will occupy is not known?  So theoretically the limit of how long a CVFR sequence can hold sync would be governmed by the number type encoding the tick count and the hz of the tick?

Reply:
As of today, that is the way all the CVFR system I know of work. So yes, the limit of how long a CVFR sequence can hold sync would be governmed by the number type encoding the tick count and the hz of the tick?